python/lsst/ts/salobj/configurable_csc.py

```diff
@@ -158,10 +158,7 @@ def __init__(
             config_dir = self.get_default_config_dir()
         else:
             config_dir = pathlib.Path(config_dir)
-            if not config_dir.is_dir():
-                raise ValueError(
-                    f"config_dir={config_dir} does not exists or is not a directory"
-                )
+
```

Comment:
Please restore this code. This catches a common error early, before anyone tries to configure the CSC.

Reply:
Please restore this code.